site/source/docs/api_reference/emscripten.h.rst

```diff
@@ -1410,6 +1414,67 @@ Functions
 
   .. note:: This requires building with ``-s ASYNCIFY_LAZY_LOAD_CODE``.
 
+.. c:function:: emscripten_fiber emscripten_fiber_create(em_arg_callback_func func, void *arg, void *stack, int stack_size)
```

Comment:
create -> init?

Reply:
I haven't updated the documentation at all yet, sorry about that.